application/lib/registration.go

```diff
@@ -84,11 +85,13 @@ type RegistrationManager struct {
 	registeredDecoys *RegisteredDecoys
 	Logger           *log.Logger
 	PhantomSelector  *PhantomIPSelector
+	LivenessTester	 *lt.LivenessTester
```

Comment:
Because this is an interface we don't need the indirection because the function implementations that fill out the interface are implemented for the pointer to `UncachedLivenessTester` for example so it can store a `*UncachedLivenessTester` in this space.
```go
LivenessTester lt.LivenessTester
```